client/src/components/productList/OrderFilter/index.tsx

```diff
@@ -1,10 +1,41 @@
-import React, { FC } from 'react';
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable react/no-array-index-key */
+
+import React, { useContext } from 'react';
+import { FilterContext } from '~/pages/ProductList';
+import { setOrder } from '~/stores/productListModule';
+import { OrderFilterWrapper, OrderHeder, OrderContainer } from './index.style';
+
+import { RefreshSVG } from '~/assets';
+
+import OrderItem from '../OrderItem';
+
+const orders = [
+  { order: 'recent', content: '최신순' },
+  { order: 'low-price', content: '낮은가격순' },
+  { order: 'high-price', content: '높은가격순' },
+];
+
+const OrderFilter: React.FC = () => {
+  const { dispatch, ...currentState } = useContext(FilterContext);
+
+  const handleResetBtnClick = () => {
+    dispatch(setOrder('recent'));
+  };
 
-const OrderFilter: FC = () => {
   return (
-    <div>
-      <span>순서 컴포넌트</span>
-    </div>
+    <OrderFilterWrapper>
+      <OrderHeder>
+        <div>보고 싶은 순서</div>
+        <img onClick={handleResetBtnClick} src={RefreshSVG} alt="reset" />
+      </OrderHeder>
+      <OrderContainer>
+        {orders.map((item, index) => (
+          <OrderItem key={index} order={item.order} content={item.content} />
+        ))}
```

Comment:
item.order가 유니크해서 키값으로 사용할 수 있어 보입니다~
리액트는 배열의 index를 키 값으로 쓰는 것을 권장하지 않더라고요

```suggestion
        {orders.map((item) => (
          <OrderItem key={item.order} order={item.order} content={item.content} />
        ))}
```

Reply:
오오 저는 왜 넘버만 가능하다고 생각했을까요. 

```
eslint-disable react/no-array-index-key

이 부분도 뺄 수 있겠네요. 감사합니다.
```